pkg/engine/jmespath/functions.go

```diff
@@ -628,6 +637,16 @@ func jpTimeSince(arguments []interface{}) (interface{}, error) {
 	return t2.Sub(t1).String(), nil
 }
 
+func jpPathCanonicalize(arguments []interface{}) (interface{}, error) {
+	var err error
+	str, err := validateArg(pathCanonicalize, arguments, 0, reflect.String)
+	if err != nil {
+		return nil, err
+	}
+
+	return filepath.Join("", str.String()), nil
```

Comment:
Is `""` really necessary here? I haven't tried it myself but the `filepath.Join` godoc suggests that it should work with a single input.

Reply:
No, it's unnecessary. Already fixed.